src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -307,6 +318,7 @@ def test_opencl_kernel_gen_wrong_kernel():
     ''' Tests that gen_ocl method raises the proper error when the
     GOKernelSchedule does not represent a proper GOcean kernel.
     '''
+    from psyclone.psyGen import Symbol
```

Comment:
pylint says Symbol is redefined from outer scope, we have it already in the file imports

Reply:
Oops, fixed.